po/zh.po

```diff
@@ -537,22 +537,22 @@ msgstr ""
 #: src/controller.cpp:719 src/itemutils.cpp:37
 #, c-format
 msgid "Failed to open queue file: %s."
-msgstr "当打开队列文件时出错: %s"
+msgstr "当打开队列文件时出错：%s"
```

Comment:
There should be a period at the end, too. (I love how Chinese has a special character for that, that little circle looks way cooler than the dot used by the Western languages.)

Reply:
Got it, I'll make some changes later. BTW, ancient Chinese writers in the Song dynasty used to use a bigger `○` to split chapters, middle ones to split paragraphs, and small ones (which we still use today) to split sentences. They really like circles.